CONTRIBUTING.md

```diff
@@ -1,26 +1,30 @@
 # Welcome!
 
-If you are reading this document then you are interested in contributing to the Islandora CLAW. All contributions are welcome: use-cases, documentation, code, patches, bug reports, feature requests, etc. You do not need to be a programmer to speak up!
+If you are reading this document then you are interested in contributing to Islandora 8. All contributions are welcome: use-cases, documentation, code, patches, bug reports, feature requests, etc. You do not need to be a programmer to speak up!
+
+We also have an irc channel -- #islandora -- on freenode.net. Feel free to hang out there, ask questions, and help others out if you can.
```

Comment:
IRC channel is essentially dead, this line should probably be removed

Reply:
should it be replaced with slack or something?

Reply:
Not yet. We're working on auto-invite solutions, but we don't know what that link will look lie yet, so there's no standard text to include.